internal/services/compute/linux_virtual_machine_scale_set_extensions_test.go

```diff
@@ -402,6 +402,7 @@ resource "azurerm_linux_virtual_machine_scale_set" "test" {
     type                       = "CustomScript"
     type_handler_version       = "2.0"
     auto_upgrade_minor_version = true
+    automatic_upgrade_enabled  = true
```

Comment:
rather than adding this to all of the existing tests - since this behaviour is optional we should add a specific test for this field, so can we revert this and add a specific test instead?

Reply:
I'm not so familiar with go or test infrastructure here, so it might be problematic.